Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -63,28 +63,28 @@ void MemoryViewWidget::UpdateFont()
   Update();
 }
 
-static int GetColumnCount(MemoryViewWidget::Type type)
+static int GetByteLength(MemoryViewWidget::Type type)
```

Comment:
A `byte` or rather an [octet](https://en.wikipedia.org/wiki/Byte) is always 8 bits so "byte length" doesn't really make sense in this context. A better wording would be type length or type size, I guess.

Reply:
ByteCount? I think it needs the word byte in there to make the value returned obvious, as bits are referenced often..

Reply:
Often in C/C++ you'll find `size`/`length`/`sizeof` referring to the size in bytes. Wording is hard and I don't I have a better name on top of my head right now...

Reply:
Oops, I forgot to do this, I'll include it in the next update.